test/extended/builds/cluster_config.go

```diff
@@ -114,6 +115,38 @@ var _ = g.Describe("[sig-builds][Feature:Builds][Serial][Slow][Disruptive] alter
 			o.Expect(ok).To(o.BeTrue())
 			return build
 		}
+		checkPodResource = func(containers []v1.Container, resources configv1.BuildDefaults) {
+			defaultResources := resources.Resources
+			o.Expect(containers).NotTo(o.BeNil())
+			foundLimitCpu := false
+			foundLimitMem := false
+			foundRequestCpu := false
+			foundRequestMem := false
+			for _, container := range containers {
+				o.Expect(container.Resources).NotTo(o.BeNil())
+				for name, value := range defaultResources.Limits {
+					switch {
+					case name == "cpu" && container.Resources.Limits[v1.ResourceName(name)] == value:
+						foundLimitCpu = true
+					case name == "memory" && container.Resources.Limits[v1.ResourceName(name)] == value:
+						foundLimitMem = true
+					}
```

Comment:
Use `if` blocks instead of `switch`. I would also separately check that the value meets what we expect. Ex:

```
if name == "cpu" {
  foundLimitCpu = true
  o.Expect(container.Resources.Limits[v1.ResourceName(name)]).To(o.Equal(value))
}

Reply:
Updated using if blocks.